packages/plugin-http-client/src/rest/mapper/DefaultToArray.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseMapper } from './BaseMapper';
+
+/**
+ * DefaultToArray mapper defines for entity property default value as empty array.
+ */
+export class DefaultToArray extends BaseMapper {
+  deserialize(value: any) {
```

Comment:
detail, ale tady by teoreticky mohlo být `deserialize(value?: any[]): any[]`? Řekl bych totiž, že teď je return type `any`, i když by to mělo být vždy nějaké pole

jako už to tak klidně nech, k tomu se stejně napřímo nepřistupje tak to je jedno :D

Reply:
asi to tak necham, at do toho nezabřednu :D